### ISSUE TITLE
Can't Create New User After Installing

### ISSUE THREAD

**Issue by swimminghiker** [microsoft/WSL#4298]
Please use the following bug reporting template to help produce issues which are actionable and reproducible, including **all** command-line steps necessary to induce the failure condition. Please fill out **all** the fields! Issues with missing or incomplete issue templates will be closed.

If you have a feature request, please post to [the UserVoice](https://wpdev.uservoice.com/forums/266908).

If this is a console issue (a problem with layout, rendering, colors, etc.), please post to the [console issue tracker](https://github.com/microsoft/console/issues).

**Important: Do not open GitHub issues for Windows crashes (BSODs) or security issues.**  Please direct all Windows crashes and security issues to secure@microsoft.com.  Ideally, please [configure your machine to capture minidumps](https://support.microsoft.com/en-us/help/315263/how-to-read-the-small-memory-dump-file-that-is-created-by-windows-if-a), repro the issue, and send the minidump from "C:\Windows\minidump\".

**Please fill out the below information:**
* Your Windows build number:  (Type `ver` at a Windows Command Prompt)
Microsoft Windows [Version 10.0.17134.829]

* What you're doing and what's happening:  (Copy&paste the full set of _specific_ command-line steps necessary to reproduce the behavior, and their output. Include screen shots if that helps demonstrate the problem.)
I ran command"Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux". Restarted Windows. Confirmed Linux is installed. Then I downloaded ubuntu and open it up. It kept asking me for a new user name. I put in several but got rejected each time. 

* What's wrong / what should be happening instead:
Should have accepted the new user name on the first try. But this is what I am getting:

Installing, this may take a few minutes...
Please create a default UNIX user account. The username does not need to match your Windows username.
For more information visit: https://aka.ms/wslusers
Enter new UNIX username: hiker
groupadd: /etc/group.5: lock file already used
groupadd: cannot lock /etc/group; try again later.
adduser: `/usr/sbin/groupadd -g 1000 hiker' returned error code 10. Exiting.
Enter new UNIX username: thchen
groupadd: lock /etc/group.lock already used by PID 5
groupadd: cannot lock /etc/group; try again later.
adduser: `/usr/sbin/groupadd -g 1000 thchen' returned error code 10. Exiting.
Enter new UNIX username:

* Strace of the failing command, if applicable:  (If `some_command` is failing, then run `strace -o some_command.strace -f some_command some_args`, and link the contents of `some_command.strace` in a [gist](https://gist.github.com/) here).

* For WSL launch issues, please [collect detailed logs](https://github.com/Microsoft/WSL/blob/master/CONTRIBUTING.md#8-detailed-logs).

See [our contributing instructions](https://github.com/Microsoft/WSL/blob/master/CONTRIBUTING.md) for assistance.

**Comment by swimminghiker** [comment]
I have uninstalled / reinstall three times already. And each time, I tried using different Linux Distribution (Ubuntu, Debian etc) but always got the same error. I can't produce any trace since it keep asking me for a new user name.

**Comment by mscraigloewen** [comment]
Here's an issue where users had the same problem which might help you: #1704

Are you using Arconis True Image?

**Comment by swimminghiker** [comment]
I searched for "acronis_drive.exe" on my PC but did not find it. I also looked into the Task Manager background processes but did not see anything from Acronis.

**Comment by swimminghiker** [comment]
Will give this a try later today: https://superuser.com/questions/1211201/wsl-bash-on-windows-10-creating-unix-user-failed-returned-error-code-10/1367878#1367878

**Comment by mattmccarty** [comment]
Has anyone found a work around for this? I have tried everything in the SO link that @swimminghiker posted above and I still can't get it to work with my AV disabled. This is the 2nd company laptop that I have tried to install WSL on in two weeks. Both have the same issue.

Windows build: 17134. 

I created a user manually by editing the user related files in /etc and created the home directory. This worked, however, when installing packages through aptitude which creates a user/group on install, the installations failed due to the same reason.
